components/cam/src/physics/cam/micro_p3_interface.F90

```diff
@@ -264,7 +273,7 @@ subroutine micro_p3_register()
  
    !! module clubb_intr (AaronDonahue: I think these are for MG only.  Should
    !  we remove?  If so, then we will have to make changes to CLUBB interface.
-   call pbuf_add_field('RELVAR',     'global',dtype_r8,(/pcols,pver/), relvar_idx)
+   call pbuf_add_field('QC_RELVAR',     'global',dtype_r8,(/pcols,pver/), qc_relvar_idx)
```

Comment:
I see the merits in making this variable name more descriptive.  My only question is, how will this impact compatibility of P3 with CLUBB.  Does CLUBB expect the PBUF variable to be called "relvar"?